src/Components/animation/AnimationEditor.js

```diff
@@ -309,6 +307,39 @@ class AnimationEditor extends React.Component {
       </View>
     );
   }
+
+  /** Used to update the animation when an animation is opened in AnimationEditorPage */
+  static getDerivedStateFromProps(props, state) {
```

Comment:
I'd like to think about a way not to use this function

Reply:
Yes I know. I used it since I wanted to make it quick and I don't know how to do it another way.